### ISSUE TITLE
CoffeeScript version?

### ISSUE THREAD

**Issue by mcginty** [robrighter/node-boilerplate#23]
Anybody interested in a port of this to CoffeeScript? I think it's a step in the right direction for readability with JavaScript. (Let me know if something like this exists already, but I really like the simplicity of this framework.)

**Comment by cjroebuck** [comment]
I'd be interested in a Coffeescript port

**Comment by mcginty** [comment]
Yeah, I forked for Coffeescript mods, so check out https://github.com/mcginty/node-boilerplate and see what you like/dislike about it so far. Request pulls if you have any contributions. I'd love to collaborate with whoever's interested.
